modules/op-running-a-pipeline.adoc

```diff
@@ -19,7 +19,7 @@ $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=pipelines-vote-api \
     -p git-url=https://github.com/openshift/pipelines-vote-api.git \
-    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
+    -p IMAGE='image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-api' \
```

Comment:
It looks like similar instances in the `op-` topics also don't use these tick marks:

```
Results for "IMAGE=" in "openshift-docs/op*.adoc"

/home/rdlugyhe/openshift-docs/modules/op-release-notes-1-5.adoc:
  134:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \


/home/rdlugyhe/openshift-docs/modules/op-running-a-pipeline.adoc:
  22:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
  45:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-ui


/home/rdlugyhe/openshift-docs/modules/op-using-tekton-chains-to-sign-and-verify-image-and-provenance.adoc:
  88:  $ tkn task start --param IMAGE=$REGISTRY/kaniko-chains --use-param-defaults --workspace name=source,emptyDir
```
Do you want to update some of these other instances?

Reply:
Do you want to update other instances of `pipelines-tutorial`?

```
Results for "pipelines-tutorial/" in "openshift-docs/op*.adoc"

/home/rdlugyhe/openshift-docs/modules/op-about-pipelinerun.adoc:
  30:      value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api


/home/rdlugyhe/openshift-docs/modules/op-about-triggers.adoc:
  78:  value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/$(tt.params.git-repo-name)


/home/rdlugyhe/openshift-docs/modules/op-adding-triggers.adoc:
  42:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/01_binding.yaml
  105:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/02_template.yaml
  135:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/03_trigger.yaml
  191:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/04_event_listener.yaml


/home/rdlugyhe/openshift-docs/modules/op-assembling-a-pipeline.adoc:
  111:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/04_pipeline.yaml


/home/rdlugyhe/openshift-docs/modules/op-creating-pipeline-tasks.adoc:
  16:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/01_apply_manifest_task.yaml
  17:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/02_update_deployment_task.yaml


/home/rdlugyhe/openshift-docs/modules/op-defining-and-creating-pipelineresources.adoc:
  42:  Enter a value for url : image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/ui:latest
  74:  Enter a value for url : image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/api:latest
  96:  api-image   image   url: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/api:latest
  97:  ui-image    image   url: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/ui:latest


/home/rdlugyhe/openshift-docs/modules/op-release-notes-1-5.adoc:
  131:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml
  134:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \


/home/rdlugyhe/openshift-docs/modules/op-running-a-pipeline.adoc:
  19:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml
  22:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
  42:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml


/home/rdlugyhe/openshift-docs/modules/op-sample-eventlistener-resource.adoc:
  13:  create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
  20:  -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/02_template.yaml
  27:  create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_trigger.yaml
  42:  https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/04_event_listener.yaml

```

Reply:
Or maybe update other instances of `pipelines-tutorial/pipelines-vote-api`?:
```
Results for "pipelines-tutorial/pipelines-vote-api" in "openshift-docs/op*.adoc"

/home/rdlugyhe/openshift-docs/modules/op-release-notes-1-5.adoc:
  134:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \


/home/rdlugyhe/openshift-docs/modules/op-running-a-pipeline.adoc:
  22:      -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \

```

Reply:
op-running-a-pipeline.adoc would be nice, IMHO release notes for 1.5 are not worth the hassle

Reply:
Thank you for your suggestions, @ppitonak. Just to ensure I understand this correctly, we are good with the changes we currently have. And updating the release notes is not required.

@Preeticp Could you please let us know your thoughts?

Reply:
Team, since this is just a minor update, I feel I'll go ahead and update the release notes for v1.9, v1.8, v1.7.

Respectfully,
Gaurav

Reply:
Team, I checked and found that the release notes v1.9, v1.8, v1.7, and v1.6 do not have any such commands, only v1.5 does. Since we are phasing out v4.8, I don't think we should update the release notes for v1.5.

Plus, to confirm my understanding I discussed the same with @sounix000 

@rolfedh If you are satisfied with the findings, I'd like to mark this chat resolved and close the reopened Jira.

Respectfully,
Gaurav

Reply:
Thanks for looking into this. Your findings make sense. 
Some remaining questions:
- For the documented commands to work, do we need to replace any other instances of `pipelines-tutorial` with `context.pipelineRun.namespace`?
- Does the enter `context.pipelineRun.namespace` or is that [a user-replaced value](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values)? If so, the openshift-docs guidelines are to enclose it with angle brackets. For example `<context.pipelineRun.namespace>`.

Reply:
You're welcome to resolve the conversation and close the reopened Jira when you believe our docs give users the correct syntax.

Reply:
Thank you for your questions.
@ppitonak @piyush-garg Could you please help us here with your thoughts?

From what I little I know, they are not user-replaceable. The $(whatever)  will simply dynamically collect the value of whatever we have in the command's runtime.

Reply:
Hi @rolfedh 
I've discussed this with @piyush-garg 

- No updates are required
- they are not user-replaceable. The $(whatever) will simply dynamically collect the value of whatever we have in the command's runtime.

Please let me your questions/LGTM.

Reply:
Users can copy&paste the command from the doc and it will be replaced by Pipelines dynamically in runtime.